app/flatpak-builtins-install.c

```diff
@@ -249,6 +253,8 @@ install_from (FlatpakDir *dir,
   flatpak_transaction_set_disable_auto_pin (transaction, opt_no_auto_pin);
   flatpak_transaction_set_reinstall (transaction, opt_reinstall);
   flatpak_transaction_set_default_arch (transaction, opt_arch);
+  flatpak_transaction_set_auto_install_sdk (transaction, opt_include_sdk);
+  flatpak_transaction_set_auto_install_debug (transaction, opt_include_debug);
```

Comment:
I think we want these two lines in `install_bundle()` too, since we still pull in dependencies from remote(s) when installing from a bundle.